app/assets/javascripts/Components/Modals/collect_submissions_modal.jsx

```diff
@@ -96,15 +102,45 @@ class CollectSubmissionsModal extends React.Component {
               <legend>{I18n.t("submissions.collect.collection_options")}</legend>
               <p>
                 <label>
-                  <input type="checkbox" name="override" onChange={this.handleOverrideChange} />
-                  &nbsp;
-                  <span
-                    dangerouslySetInnerHTML={{
-                      __html: I18n.t("submissions.collect.override_existing_html"),
-                    }}
+                  <input
+                    type="checkbox"
+                    defaultChecked={this.state.override}
+                    name="override"
+                    data-testid="chk_recollect_existing_submissions"
+                    onChange={this.handleOverrideChange}
                   />
+                  &nbsp;
+                  <span data-testid="lbl_recollect_existing_submissions">
+                    {I18n.t("submissions.collect.override_existing")}
+                  </span>
                 </label>
               </p>
+              {this.state.override && (
+                <p style={{marginLeft: "15px"}}>
+                  <label>
+                    <input
+                      type="checkbox"
+                      defaultChecked={this.state.retain_existing_grading}
+                      name="retain_existing_grading"
+                      data-testid="chk_retain_existing_grading"
+                      onChange={this.handleRetainExistingGradingChange}
+                    />
+                  </label>
+                  &nbsp;
+                  {this.state.retain_existing_grading ? (
```

Comment:
Keep the checkbox label the same for both checked and unchecked states. Instead, if the option is unchecked, show a warning div (same styling as warning flash messages) with the extra warning message. Since the div will have good styling already, you don't need to bold the warning message.